nixos/modules/services/networking/unifi.nix

```
@@ -28,10 +28,36 @@ let
     }
   ];
   systemdMountPoints = map (m: "${utils.escapeSystemdPath m.where}.mount") mountPoints;
+
+  siteRules = concatLists (mapAttrsToList (siteId: site:
+    let gwCfg = site.configGateway;
+        gwCfgFile = pkgs.writeText "config.gateway.json" (if isAttrs gwCfg
+                                                          then builtins.toJSON gwCfg
+                                                          else gwCfg);
+    in
+    [
+        "d '${stateDir}/data/sites/${siteId}' 0700 unifi - - -"
+        "L+ '${stateDir}/data/sites/${siteId}/config.gateway.json' - - - - ${gwCfgFile}"
+    ]
+  ) cfg.sites);
 in
 {
 
   options = {
+    services.unifi.sites = mkOption {
+      description = "Site configuration";
+      default = {};
```

maybe provide a simple example (this one is from the linked ui.com document)
```suggestion
      default = {};
      example = { default = { system.conntrack.timeout.tcp.close = "20"; }; };
```